.github/workflows/test-and-deploy.yml

```diff
@@ -28,11 +27,41 @@ jobs:
         npm test
         serverless --version
         serverless info --conceal
+
+  deploy_us_east_1:
+    runs-on: ubuntu-latest
+    timeout-minutes: ${{ fromJSON(vars.DEFAULT_JOB_TIMEOUT_MINUTES) }}
+    needs: test
+    steps:
+    - uses: actions/checkout@v4
+    - name: Use Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '18.x'
+    - name: install-serverless
+      run: npm install --no-fund -g serverless@3.25
```

Comment:
Finally one outside of a deploy script, so could use serverless github action, but I am hesitant about that as it is not maintained currently, but still seems up to date.

Reply:
Advantage: someone else maintains the `npm install` command
Disadvantage: one more dependency with all of the associated risks

I think I'll keep it as is. Thanks.